src/main/java/seedu/allonus/modules/StudyManager.java

```diff
@@ -180,37 +178,42 @@ private void printAlreadyInStudyManagerMessage(TextUi ui) {
     }
 
     private void openIcsFile(TextUi ui, ModuleCalendarReader icsParser) {
-        printMessage("Please enter the name of your .ics file from nusmods:");
+        printMessageWithDivider("Please enter the name of your .ics file from nusmods:");
         String input = ui.getUserInput();
         ArrayList<Module> icsModulesList = icsParser.readIcsFile(input);
         if (icsModulesList != null) {
             modulesList.addAll(icsModulesList);
             isModified = true;
         }
-        printMessage("Exiting read ics mode");
+        printMessageWithDivider("Exiting read ics mode");
     }
 
     /**
      * Prints a given input string using system output.
      * @param message String that is to be printed on the console.
      */
+    public static void printMessageWithDivider(String message) {
+        textUi.showToUser(message);
+        //System.out.println(message);
+    }
+
     private void printMessage(String message) {
         System.out.println(message);
     }
 
     private void printWelcomeMessage() {
-        printMessage(WELCOME_MESSAGE);
+        printMessageWithDivider(WELCOME_MESSAGE);
```

Comment:
Any reason not to use textui.showToUser directly?

Reply:
It was easier to refactor an existing print method I had that had the same function, as it was already being used.
Also wanted the distinction to be clear between printing a normal message without divider and printing with divider!